pkg/webhooks/jobset_webhook_test.go

```diff
@@ -35,6 +35,8 @@ var TestPodTemplate = corev1.PodTemplateSpec{
 	},
 }
 
+var defaultNetwork = &jobset.Network{EnableDNSHostnames: ptr.To(true), PublishNotReadyAddresses: ptr.To(true)}
```

Comment:
Is this only used in `TestJobSetDefaulting`? If so let's move it into TestJobSetDefaulting next to the defaultSuccessPolicy and defaultStartupPolicy variables.

Reply:
done.